lib/zigbee/ZigBeeDevice.js

```diff
@@ -73,6 +73,7 @@ class ZigBeeDevice extends MeshDevice {
 
 		// Start listening for incoming reports
 		this.node.on('report', report => {
+			this._debug('report', report);
```

Comment:
Still needed?

Reply:
Yes, it is conditional and helps during development.